circularSeekBar/src/main/java/me/tankery/lib/circularseekbar/CircularSeekBar.kt

```diff
@@ -705,6 +747,12 @@ class CircularSeekBar @JvmOverloads constructor(
         pointerPaint.strokeJoin = Paint.Join.ROUND
         pointerPaint.strokeCap = circleStyle
 
+        pointerFillPaint.set(pointerPaint)
+        pointerFillPaint.color = pointerFillColor
+        pointerFillPaint.style = Paint.Style.FILL_AND_STROKE
```

Comment:
This I believe should be `Paint.Style.STROKE`, otherwise the view will have visual issue when the `pointerAngle` > 0.